marlowe-runtime/sync-api/Language/Marlowe/Protocol/Query/Server.hs

```diff
@@ -32,6 +33,7 @@ marloweQueryServer getContractHeaders getContractState getTransaction = go
     serviceRequest :: Request a -> m a
     serviceRequest = \case
       ReqContractHeaders range -> getContractHeaders range
-      ReqContractState range -> getContractState range
-      ReqTransaction range -> getTransaction range
+      ReqContractState contractId -> getContractState contractId
```

Comment:
So the API changed. I agree it is cleaner this way.

Reply:
No, just some misnamed variables

Reply:
Ah